static_report/src/components/ComparisonReport/CRList.tsx

```diff
@@ -96,27 +102,63 @@ export function ComparisonReportList({ data }) {
               <Thead>
                 <Tr>
                   <Th>Name</Th>
-                  <Th>Passed</Th>
-                  <Th>Failed</Th>
+                  <Th>PipeRider</Th>
+                  <Th>dbt</Th>
                   <Th>Rows</Th>
                   <Th>Columns</Th>
                 </Tr>
+                <Tr>
+                  <Th />
+                  <Th>
+                    <Text as="span" mr={8}>
+                      Passed
+                    </Text>
+                    <Text as="span">Failed</Text>
+                  </Th>
+                  <Th>
+                    <Text as="span" mr={8}>
+                      Passed
+                    </Text>
+                    <Text as="span">Failed</Text>
+                  </Th>
+                  <Th />
+                  <Th />
+                </Tr>
               </Thead>
               <Tbody>
                 {Object.keys(tables).map((key) => {
                   const table = tables[key];
-                  const baseOverview = getReportAsserationStatusCounts(
-                    // TODO: upstream types can improve
-                    table.base?.assertion_results
+
+                  const baseOverview = getComparisonAssertionTests({
```

Comment:
I think all the `*Overview` assignments here can be refactored as an iterable, rather than called 4 times